htmd/parameterization/test.py

```diff
@@ -395,6 +395,18 @@ def test_h2o2_dihed_opt_mm_fake(self):
         self._run(refDir, resDir, 'parameterize input.mol2 -f GAFF2 --charge-type Gasteiger --min-type mm --scan-type mm --fake-qm')
         self._test(refDir, resDir)
 
+    def test_glycol_dihed_opt_mm_fake(self):
+        refDir = os.path.join(self.dataDir, 'glycol_dihed_opt_mm_fake')
+        resDir = os.path.join(self.testDir, 'glycol_dihed_opt_mm_fake')
+        self._run(refDir, resDir, 'parameterize input.mol2 -f GAFF2 --charge-type Gasteiger --min-type None --scan-type mm --fake-qm --dihed-num-searches 0')
+        self._test(refDir, resDir, energyProfileAbsTol=0.01)
```

Comment:
How is it possible: energy needs a large tolerance, while the rest number pass with low tolerances

Reply:
There is no fitting done so I assume it will be pretty exact. Anyway, I actually think 0.01 should be default on the tests since the files are written with that precision and any kind of noise could flip numbers that are on the 0.005 limit

Reply:
Which files do you refer to?

Reply:
The .dat files of the plots. Ah sorry ok I missed a zero it seems. So 0.001 should be the tolerance in my opinion.

Reply:
0.002 actually since it needs to be lower-than

Reply:
Changed it to 0.002 now.

Reply:
OK!